src/main/java/duke/Event.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * One of the tasks that could be created by user.
+ */
+public class Event extends Task {
+    protected LocalDate at;
+
+    /**
+     * Event constructor.
+     *
+     * @param description Describes the task.
```

Comment:
Remember to use "Description" instead of the lower case form for the first word in your Javadoc.